### ISSUE TITLE
V10: Don't make `inBuffer` private

### ISSUE THREAD

**Issue by caewok** [foundryvtt/foundryvtt#7272]
Foundry 10.270 sets `inBuffer` to be a private property of `CanvasVisibility.` The problem is that `testVisibility` uses `inBuffer` to test for points inside the buffer region. Thus, in order to modify `testVisibility`, one would first have to re-do the test , in `CanvasVisibility.prototype.refresh`, of all the vision sources to determine whether `inBuffer` should be true or false. 

In addition, it seems like it might actually be useful to be able to easily test, by checking the `inBuffer` variable, whether there is a source of vision outside the primary scene canvas. 

### User Experience

No change from the user perspective.

### Priority/Importance

Low, but seems like an easy change. Obviously I don't know the reasoning behind making this private, but it doesn't seem like anything too terrible could happen even if someone were able to randomly set this property. In the meantime, it is possible to work around it by re-running the test for `inBuffer` after each refresh and storing the result somewhere.

**Comment by aaclayton** [comment]
Makes sense, will make this change. Related to https://github.com/foundryvtt/foundryvtt/issues/7274
